### PR TITLE
✨ Frontend TIP Feature: Pop up window with available new Plans

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

For the coming TIP v3, 3 more templates (3 more plus buttons) are required.

Instead of showing 6 plus buttons, only one New Plan button we will be shown and then the users will need to choose what Plan they want to start.

**Bonus**:
Plus buttons information lives now in a separate file

SelectNewPlan

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by pcrespov — COMMENTED** [review]
:+1:

File: services/static-webserver/client/source/class/osparc/dashboard/StudyBrowser.js [hunk 1]
```diff
@@ -47,28 +47,48 @@ qx.Class.define("osparc.dashboard.StudyBrowser", {
         title: "Classic TI",
         description: "Start new TI planning",
         newStudyLabel: "Classic TI",
+        category: "precomputed",
         idToWidget: "newTIPlanButton"
       },
       "mTI": {
         templateLabel: "mcTI Planning Tool",
         title: "Multichannel TI",
         description: "Start new mcTI planning",
```
Comment (elisabettai) [inline services/static-webserver/client/source/class/osparc/dashboard/StudyBrowser.js]: Maybe spell it out, so "Start new Multichannel TI planning"

File: services/static-webserver/client/source/class/osparc/dashboard/StudyBrowser.js [hunk 2]
```diff
@@ -47,28 +47,48 @@ qx.Class.define("osparc.dashboard.StudyBrowser", {
...
         newStudyLabel: "Multichannel TI",
+        category: "precomputed",
         idToWidget: "newMTIPlanButton"
       },
       "pmTI": {
         templateLabel: "pmTI Planning Tool",
         title: "Phase-Modulation TI",
         description: "Start new pmTI planning",
```
Comment (elisabettai) [inline services/static-webserver/client/source/class/osparc/dashboard/StudyBrowser.js]: "Start new Phase-Modulation TI planning"

**Review by elisabettai — APPROVED** [review]
Very nice! 
I've just added some comments:
- Spell out multichannel/phasemodulation everywhere (or not): just use the same convention on all the buttons
- "plan" or "planning" use the same word everywhere. I'd go for plan as it is shorter, but maybe @Konohana0608 has a different opinion

**Comment by odeimaiz** [comment]
@elisabettai Wording was reviewed and approved by @Konohana0608, but I switched to Multichannel and Phase-Modulation.

File: services/static-webserver/client/source/resource/osparc/new_studies.json [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      "expectedTemplateLabel": "pmTI Planning Tool",
+      "title": "Phase-Modulation TI",
+      "description": "Start new Phase-Modulation TI planning",
+      "newStudyLabel": "Phase-Modulation TI",
+      "category": "precomputed",
+      "idToWidget": "newPMTIPlanButton"
+    }, {
+      "expectedTemplateLabel": "personalized TI Planning Tool",
+      "title": "Personalized Classic TI",
+      "description": "Start new Personalized Classic TI plan",
```
Comment (elisabettai) [inline services/static-webserver/client/source/resource/osparc/new_studies.json:28]: Here you're using "plan", in the other "planning"

**Review by sanderegg — APPROVED** [review]

**Review by ignapas — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]

**Pull request merged**